src/Authentication/Passwords/NothingPersonalValidator.php

```diff
@@ -52,8 +52,8 @@ public function check(string $password, ?User $user = null): Result
      */
     protected function isNotPersonal(string $password, ?User $user): bool
     {
-        $userName = \strtolower($user->username);
-        $email    = \strtolower($user->email);
+        $userName = strtolower($user->username ?? '');
+        $email    = strtolower($user->email);
```

Comment:
Should we anticipate users trying to work without emails?
```suggestion
        $email    = strtolower($user->email ?? '');
```

Reply:
It is out of scope of this PR.

The current implementation requires email. If we support users without email, I guess more to work.